server.py

```diff
@@ -1,78 +1,83 @@
 import requests
-# from flask import Flask
-from flask import Flask, request,send_file,jsonify
-from timetable import build_courses, generate_ics
+from flask import Flask, request, send_file, jsonify
 import io
-
+import jwt
 from flask_cors import CORS
+from datetime import datetime, timedelta
+from timetable import build_courses, generate_ics
 from session_manager import SessionManager
+from utils import ERPSession
 
 app = Flask(__name__)
 CORS(app, support_credentials=True)
 
+# Initialize the SessionManager instance
+session_manager = SessionManager()
 
-# Start the first cleanup task
-SessionManager.cleanup_sessions()
-
-
-
-@app.route("/secret-question", methods=["POST"])
-def get_secret_question():
+@app.route("/create-session", methods=["POST"])
+def create_session():
     try:
         data = request.form
         roll = data.get("roll")
         if not roll:
             return jsonify({"status": "error", "message": "Missing roll number"}), 400
         
-        secret_question = SessionManager.get_secret_question(roll)
-        return jsonify({"status": "success", "secret_question": secret_question}), 200
+        token = session_manager.create_session(roll)
+        return jsonify({"status": "success", "token": token}), 200
     except Exception as e:
         return jsonify({"status": "error", "message": str(e)}), 500
 
+@app.route("/secret-question", methods=["POST"])
+def get_secret_question():
```

Comment:
don't make it two different endpoints.
Create session, and send token and security question in the same endpoint